decidim-admin/app/commands/decidim/admin/update_help_sections.rb

```diff
@@ -5,22 +5,36 @@ module Admin
     # This command gets called when a content block is updated from the admin
     # panel.
     class UpdateHelpSections < Decidim::Command
-      def initialize(form, organization)
+      def initialize(form, organization, user)
         @form = form
         @organization = organization
+        @user = user
       end
 
       def call
         return broadcast(:invalid) unless @form.valid?
 
         ActiveRecord::Base.transaction do
           @form.sections.each do |section|
-            ContextualHelpSection.set_content(@organization, section.id, section.content)
+            next unless content_has_changed(section)
+
+            Decidim.traceability.perform_action!("update", ContextualHelpSection, @user, { "resource" => { "title" => section.id.humanize } }) do
+              ContextualHelpSection.set_content(@organization, section.id, section.content)
+              ContextualHelpSection.find_by(organization: @organization, section_id: section.id)
```

Comment:
Is this line necessary? As far as I see, `ContextualHelpSection.set_content` will [return the content](https://github.com/decidim/decidim/blob/82c0ba5c043d3b9bde96af7c19664439ee86fdc2/decidim-core/app/models/decidim/contextual_help_section.rb#L46 )

Reply:
It will return the content, but we need the block inside  "perform_action" to return the resource, the ContextualHelpSection object